example/lib/src/bloc/ApplicationBloc.dart

```diff
@@ -202,7 +202,9 @@ class ApplicationBloc extends BlocBase {
   void _fetchSongData({String query}) {
     if (query == null)
       audioQuery
-          .getSongs(sortType: _songSortTypeSelected)
+          .getSongs(
+              sortType: _songSortTypeSelected,
+              storageType: StorageType.INTERNAL)
```

Comment:
@sc4v3ng3r I did a simple test with an example source. How do you apply it to artists, albums, or playlists?